app/controllers/timeline_entries_controller.rb

```diff
@@ -7,9 +7,27 @@ def new
     @timeline_entry = coronavirus_page.timeline_entries.new
   end
 
+  def create
+    @timeline_entry = coronavirus_page.timeline_entries.new(timeline_entry_params)
+
+    if @timeline_entry.save && draft_updater.send
+      redirect_to coronavirus_page_path(coronavirus_page.slug), notice: "Timeline entry was successfully created."
+    else
+      render :new
```

Comment:
this should probably have a status of unprocessable entity to indicate that validation failed - but maybe don't bother if it's a nightmare for consistency with the rest of the app.